lib/hex_web/docs_tar.ex

```diff
@@ -0,0 +1,56 @@
+defmodule HexWeb.DocsTar do
```

Comment:
the idea was to extract logic from the controller, similarly how we have HexWeb.Tar.

btw, wdyt about renaming HexWeb.Tar to HexWeb.ReleaseTar? Or: HexWeb.Tar.Release and HexWeb.Tar.Docs